tests/test_jdate.py

```diff
@@ -137,3 +137,12 @@ def test_unpickle_older_date_object(self):
         with open('tests/pickled_objects/%s' % pickled_object_file, 'rb') as f:
             d = pickle.load(f)
         self.assertEqual(d, jdatetime.date(1400, 10, 11))
+
+    def test_fromisoformat(self):
+        jd = jdatetime.date.fromisoformat("1378-2-22")
```

Comment:
I think this is not the right test because this format is not valid in Python:

```
>>> date.fromisoformat('2022-2-01')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: Invalid isoformat string: '2022-2-01'
```

Reply:
as a user i think it's "ok" to accept this format too.
But fine i will raise an exception for that.